google/provider_test.go

```diff
@@ -53,6 +53,7 @@ func testAccPreCheck(t *testing.T) {
 		"GOOGLE_CREDENTIALS",
 		"GOOGLE_CLOUD_KEYFILE_JSON",
 		"GCLOUD_KEYFILE_JSON",
+		"GOOGLE_USE_DEFAULT_CREDENTIALS",
```

Comment:
No further changes are required, the code is already there.
providerConfigure calls loadAndValidate, which uses the DefaultClient if no credentials are supplied.